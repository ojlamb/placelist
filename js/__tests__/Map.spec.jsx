import React from "react";
import Map from "../components/map/MapPage";

test("Search renders correctly", () => {
  const component = shallow(<Map />);
  expect(component).toMatchSnapshot();
});
