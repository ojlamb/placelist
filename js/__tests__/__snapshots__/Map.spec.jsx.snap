// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<Connect(Map)
  classes={
    Object {
      "actionButton": "Connect-Map--actionButton-1",
      "icon": "Connect-Map--icon-2",
    }
  }
/>
`;
